modules/chimera/src/main/java/org/dcache/chimera/DirectoryStreamHelper.java

```diff
@@ -25,7 +25,7 @@
 
 public class DirectoryStreamHelper {
 
-    private static final Logger _log = LoggerFactory.getLogger(DirectoryStreamHelper.class);
+    private static final Logger logger = LoggerFactory.getLogger(DirectoryStreamHelper.class);
```

Comment:
If you change it, then yes. Please use LOGGER. But please make variable name only commit.